app/scripts/router.js

```diff
@@ -120,6 +120,7 @@ function (
       this.user = options.user;
       this.interTabChannel = options.interTabChannel;
       this.formPrefill = options.formPrefill;
+      this.notifications = options.notifications;
```

Comment:
router is beginning to have a ton of deps that it only uses to create views. We've previously talked about creating a ViewFactory that receives all the deps that way router can stay pretty dumb. Maybe we should revisit that idea soon (as in, not this PR).